data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,315 @@ mission "Coalition License Patch"
+			`	"Responsible for overseeing deliveries of such exotic materials to the warehouses here, I also am. For the inconvenience and bureaucracy, I apologize," he says. "Forget to properly introduce myself, I often do. Lugglop, I am called."`
+			choice
+				`	"Nice to meet you, I suppose."`
+				`	"Do you need more samples? Is that why you called me here?"`
+			`	"The harder to keep track of the samples, it becomes, the more samples there are," he says. "Notified by the Heliarchs, I was. No more samples, for now, I can request. Risk letting them be stolen, they do not want."`
+			`	He then shows you a map of the Coalition, pointing to a system in Saryd space.`
+			`	"Use more delicate procedures and research, to study the samples we must. For that, friends of mine on <destination>, waiting will be," he says. "Bring them here, would you, Captain?"`
+			choice
+				`	"Of course, they'll be here soon."`
+					accept
+				`	"Sorry, I'm not one to take passenger jobs."`
+					decline
+	on complete
+		conversation
+			`When you land on <planet>, Lugglop contacts you, and informs you that he has already told his friends about you. He says they will be waiting in the spaceport.`
+
+
+
+mission "Coalition Yottrite 5"
+	name "Saryd Specialists"
+	description "Transport the twins Nasilor and Alituri to <destination>, where they will assist in the Yottrite research."
+	passengers 2
+	to offer
+		has "Coalition Yottrite 4: done"
+	source "Shadowed Valley"
+	destination "Delve of Bloptab"
+	on offer
+		conversation
+			`A Saryd wearing their version of a lab coat is waiting for you when you enter the spaceport.`
+			`	"Working with Lugglop, you are?" He asks, and you nod. "Nasilor, my name is. Packing up our belongings, my sister is. Bring you to our laboratory, I shall."`
+			`	You follow him deep into the underground spaceport, to the "residential" section, which to you is indistinguishable from a research compound. Still on the way, the endless doors to your left and right occasionally give space for massive glass panels, giving you a nice view of the massive crystal formations here.`
+			`	Nasilor stops by a door immediately after one of these "viewing sites", and opens it. He introduces you to his sister, and you two start helping her pack what remains. Most of what they're bringing are several research papers, though you can't know what's written on them.`
+			`	Finally, you start heading back to your ship. Noticing your interest in taking in the beauty of the gypsum caves, the two Saryd researchers slow down their pace and start to tell you more about themselves. They are twins, both born on Saros, but have been working here for decades. Though you don't understand much of the technical jargon, you figure out that Nasilor has specialized in cryptocrystalline compounds, while Alituri focused her studies on planetary and cosmic conditions that lead to the formation of the gymsum caves here.`
```

Comment:
```suggestion
			`	Finally, you start heading back to your ship. Noticing your interest in the beauty of the gypsum caves, the two Saryd researchers slow down their pace and start to tell you more about themselves. They are twins, both born on Saros, but have been working here for decades. Though you don't understand much of the technical jargon, you figure out that Nasilor specializes in cryptocrystalline compounds, while Alituri focuses her studies on planetary and cosmic conditions that lead to the formation of the gypsum caves here.`
```
`in taking in the beauty` can be simplified to `in the beauty` (personal preference). Changed verb tenses , as they're scientists, not ex-scientists. Assumed `gypsum`, but is it `gymsum`?